lib/pacemaker/pcmk_sched_allocate.c

```diff
@@ -215,6 +215,25 @@ apply_stickiness(pe_resource_t *rsc, pe_working_set_t *data_set)
                       rsc->cluster);
 }
 
+/*!
+ * \internal
+ * \brief Apply shutdown locks for all resources as appropriate
+ *
+ * \param[in] data_set  Cluster working set
+ */
+static void
+apply_shutdown_locks(pe_working_set_t *data_set)
+{
+    if (pcmk_is_set(data_set->flags, pe_flag_shutdown_lock)) {
```

Comment:
If you switch this to `if (!pcmk_is_set(...))` and return, then you can un-indent everything after.

Reply:
added commit on top for this and the other similar one